### PR TITLE
Responsive flag

### PR DESCRIPTION
**Pull request by cenjennifer**
Making `Flag` component responsive

### CODE REVIEW

**Comment by jxnblk** [comment]
Thanks for this! It could be helpful if you uploaded some before and after screenshots to see what the change here looks like

**Comment by cenjennifer** [comment]
@jxnblk - Here's the screenshot. So desktop should still look as is, but when its mobile, this is what it looked like:
Before:
screen shot 2018-04-10 at 12 38 19 pm

After:
screen shot 2018-04-10 at 12 39 40 pm

File: src/Flag.js [hunk 1]
```diff
@@ -13,29 +15,25 @@ const darkBorderColor = props => {
...
   flex: none;
-  min-height: 32px;
-  margin-left: -10px;
   background-color: ${props => theme(`colors.${props.color}`)(props)};
-  border-radius: ${theme('radius')};
+  border-radius: 0 ${theme('radius')} ${theme('radius')} 0;
   /* for 32 x 8 triangle */
   transform: skew(-14deg);
+  position: relative;
+  z-index: 1;
```
Comment (msafari) [inline src/Flag.js]: n-b: Are these z-indexes gonna be problematic if there are any z-index conflicts in the app that consumes this?

Comment (msafari) [reply src/Flag.js]: We can also always update if it is an issue

Comment (cenjennifer) [reply src/Flag.js]: Great! Thanks :)

**Pull request merged**